snuba/utils/kafka/consumers/confluent.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any, Callable, Mapping, Optional, Sequence
+
+from confluent_kafka import Consumer as ConfluentConsumer
+from confluent_kafka import Message, TopicPartition
+
+from snuba.utils.kafka.consumers.abstract import Consumer as AbstractConsumer
+
+
+class Consumer(AbstractConsumer[Message]):
```

Comment:
This wrapper is needed to allow type checking the Confluent consumer.

Reply:
I think this comment would be more useful as part of the docstring.

Reply:
Right, thanks for pointing that out.

Reply:
This could be better, but is a good start, I think: https://github.com/getsentry/snuba/blob/03663445f3a4ffcf8fdbbcbe15fb4347f327461c/snuba/utils/kafka/consumers/abstract.py#L26-L37